doc/btrfs-subvolumes.md

```diff
@@ -0,0 +1,183 @@
+# BtrFS Subvolumes
+
+There are different ways to define the list of BtrFS subvolumes in YaST.
+
+## Control File and Storage Proposal
+
+The list of subvolumes for a BtrFS file system can be configured by means of the control file. For example, for SLE (and also for openSUSE) the control file contains something like the following:
+
+~~~
+<partitioning>
+  <volumes config:type="list">
+    <volume>
+      <mount_point>/</mount_point>
+      <fs_type>btrfs</fs_type>
+      <btrfs_default_subvolume>@</btrfs_default_subvolume>
+      <subvolumes config:type="list">
+        <subvolume>
+            <path>home</path>
+        </subvolume>
+        <subvolume>
+            <path>opt</path>
+        </subvolume>
+        <subvolume>
+            <path>root</path>
+        </subvolume>
+        <subvolume>
+            <path>srv</path>
+        </subvolume>
+        <subvolume>
+            <path>tmp</path>
+        </subvolume>
+      </subvolumes>
+    </volume>
+  </volumes>
+</partitioning>
+~~~
+
+This configuration of subvolumes is taken into accout by the Storage Proposal when calculating the partitioning proposal. But it also affects to the Expert Partitioner, see *Expert Partitioner* section.
+
+Note that there is a `btrfs_default_subvolume` option. This option allows to configure a kind of "prefix" subvolume, and it has some implications over the rest of subvolumes:
+
+* All the subvolumes are created as children of the default one.
```

Comment:
We should clarify that, in this context, when we say "default subvolume"  we are referring to that special prefix subvolume... which may end up being the real default subvolume or not (is very unfortunate that somebody decided to misuse a term that already existed in Btrfs).